### DISCUSSION TITLE
Generating distinct wind simulations with TurbSim without manually changing the random seed

### DISCUSSION THREAD

**Discussion by sdragonslayer** [OpenFAST/openfast#2279] [Q&A] [answered]
Hi, I'm trying to generate around 20 different .bts files from turbsim, and I wanted each of them to be different in terms of windspeed at each point in time. However, after generating around three files (no changes were made to the inp file), I realized that each time turbsim generated the exact same data. 

I set uref to 12 m/s, and I know that over the AnalysisTime the wind speed averages out to 12 m/s. However, the only way I am able to change the instantaneous wind data to be different each turbsim.bts generation is if I manually change the random seed 1 to a different value in the inp file(for example from 13428 to 13429). Is there any other way to do this, or is manually changing the only option?

I already saw this thread, https://github.com/OpenFAST/openfast/discussions/1723, but I wasn't able to get an answer from it.

**Comment by jjonkman** [comment]
Dear @sdragonslayer,

For the same TurbSim simulation set up (the same spatial-temporal resolution and meteorological boundary conditions), the way to obtain different time series is to change the random seeds (`RandSeed1` and/or `RandSeed2`).  If you are generating many instances of a given condition (as is typically the case), you are likely using a script to edit the TurbSim input file and execute TurbSim, in which case the script should change the seed between distinct simulations.

Best regards,

**Comment by sdragonslayer** [comment]
Cool, thank you so much! Also, another quick question, does the RanLux second seed not automatically do that already? Sorry, I am just not very familiar with random generation.

**Reply by jjonkman** [reply, marked as answer]
Dear @sdragonslayer,

Setting `RandSeed2` = "RanLux" tells TurbSim to use Lüscher's level 3 "Luxury Pseudorandom Numbers" generator in place of the intrinsic random number generator; in this case, you `RandSeed1` is needed to set the seed (the seed is never automatic).  See the TurbSim User's Guide for more information: https://openfast.readthedocs.io/en/main/_downloads/cb14d3e2d3533d76e405d730fea19846/TurbSim_v2.00.pdf.

Best regards

**Reply by sdragonslayer** [reply]
Thank you so much!
